docs/data/data-grid/overview/overview.md

```diff
@@ -7,119 +7,51 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/grid/
 
 # MUI X Data Grid
 
-<p class="description">A fast and extendable React data table and React data grid. It's a feature-rich component available with MIT or commercial licenses.</p>
-
-The Data Grid component is built with React and TypeScript to provide a smooth UX for manipulating an unlimited set of data.
-It features an intuitive API for real-time updates as well as theming and custom templates—all with blazing-fast performance.
+<p class="description">A fast and extensible React data table and React data grid, with filtering, sorting, aggregation, and more.</p>
 
 {{"component": "@mui/docs/ComponentLinkHeader"}}
 
 ## Overview
 
-The Data Grid presents information in a structured format of rows and columns.
-The data is displayed in a user-friendly interface for efficient editing, reviewing, and analysis.
-
-:::info
-Visit the [installation guide](/x/react-data-grid/getting-started/#installation) to learn how to install the correct package version and dependencies.
-:::
+The MUI X Data Grid is a TypeScript-based React component that presents information in a structured format of rows and columns.
+It provides developers with an intuitive API for implementing complex use cases; and end users with a smooth experience for manipulating an unlimited set of data.
 
-The component comes in three different versions, one available under MIT license and two available under commercial license.
+The Grid's theming features are designed to be frictionless when integrating with Material UI and other MUI X components, but it can also stand on its own and be customized to meet the needs of any design system.
 
-## MIT license (free forever)
+The Data Grid is **open-core**: The Community version is MIT-licensed and free forever, while more advanced features require a Pro or Premium commercial license.
 
-The MIT-licensed version (also referred to as the "Community version") is a stronger alternative to [plain data tables](/material-ui/react-table/#sorting-amp-selecting).
-It's a clean abstraction with basic features like editing, pagination, column grouping, and single-column sorting and filtering.
+See [MUI X Licensing](/x/introduction/licensing/) for complete details.
 
-```js
-import { DataGrid } from '@mui/x-data-grid';
```

Comment:
Can we bring those back? I find myself being confused when landing on https://next.mui.com/x/react-data-grid/.